lib/net/nforceif/include/lwipopts.h

```diff
@@ -277,6 +310,24 @@
  */
 #define LWIP_TCP                        1
 
+/**
+ * TCP_MSS: TCP Maximum segment size. (chosen to be the maximum
+ * that can be transmitted over IPv6+PPPoE unfragmented)
+ * For the receive side, this MSS is advertised to the remote side
+ * when opening a connection. For the transmit size, this MSS sets
+ * an upper limit on the MSS advertised by the remote host.
+ */
+#define TCP_MSS                         1432
```

Comment:
An ethernet frame is 1500 bytes. Subtracting 8 bytes overhead for PPPoE and another 40 bytes for the IPv6 header  gives us 1452 bytes. 🤔 What am I missing to match your proposed 1432 bytes?

Reply:
Oh, I forgot TCP itself, right? Can't resolve the comment myself, sorry!